experimental/middleware/middleware.go

```diff
@@ -4,11 +4,33 @@
 package experimental
 
 import (
+	"embed"
 	"net/http"
 
 	"github.com/corazawaf/coraza/v3"
 )
 
+//go:embed error_template.html error_template.html
+var embededTemplates embed.FS
+
+var (
+	errorTemplate        []byte
+	interruptionTemplate []byte
+)
+
+func init() {
+	var err error
+	errorTemplate, err = embededTemplates.ReadFile("error_template.html")
+	if err != nil {
+		panic(err)
+	}
+	interruptionTemplate, err = embededTemplates.ReadFile("interruption_template.html")
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Shouldn't we avoid error handling at the `init` level? Can we have a `readTemplates` method instead that will be called by the middleware?